src/marks/text.js

```diff
@@ -44,16 +47,17 @@ export class Text extends Mark {
     this.fontStyle = string(fontStyle);
     this.fontVariant = string(fontVariant);
     this.fontWeight = string(fontWeight);
+    this.cx = cx;
+    this.cy = cy;
+    this.anchor = anchor;
```

Comment:
We should parse and validate the anchor here in the constructor, rather than deferring validation until use.